<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>385</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="checkWindowBox">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hann</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hamming</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="sing_label">
          <property name="text">
           <string>Vector</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="singularSBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="usePeakBtn">
        <property name="text">
         <string>Peak</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="useGassiorBtn">
        <property name="text">
         <string>Gasior</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="useNaffBtn">
        <property name="styleSheet">
         <string notr="true">selection-color: rgb(0, 255, 0);</string>
        </property>
        <property name="text">
         <string>NAFF</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="log_mod">
     <property name="text">
      <string>Log</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lb_label">
       <property name="text">
        <string>Left</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="lboardSBox">
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="rb_label">
       <property name="text">
        <string>Right</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="rboardSBox">
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
